izanami-server/conf/application.conf

```diff
@@ -11,10 +11,14 @@ play.http {
 http.port=9000
 http.port=${?HTTP_PORT}
 
+base.url="/"
+base.url=${?APPLICATION_BASE_URL}
```

Comment:
This is not useful to declare this config. It was the case for the context because I was overriding a play config.